.github/workflows/flutter-build-android.yml

```diff
@@ -44,66 +44,66 @@ jobs:
   build:
     runs-on: macos-latest
```

Comment:
Deze kunnen we ook op `ubuntu-latest` zetten. Die is iets sneller en wellicht uiteindelijk ook goed koper (maar daar ben ik niet helemaal zeker van).

Reply:
Changes in flutter_build-android.yml zijn alleen door formatting.

Reply:
Maakt niet uit toch, kunnen dit als nog even verbeteren.